charts/thoras/values.yaml

```diff
@@ -53,4 +53,15 @@ thorasDashboard:
     memory: 100Mi
   port: 80
 
+thorasMonitor:
```

Comment:
I see we're referencing `.Values.thorasMonitor.imageTag` in line 29 of the deployment but there's no default value. That seems problematic -- could we set the default value to .Values.thorasVersion?